schema.sql

```diff
@@ -43,4 +44,9 @@ CREATE TABLE visits (
     vet_id INT REFERENCES vets(id),
     visit_date DATE,
     PRIMARY KEY (animal_id, vet_id, visit_date)
-);
\ No newline at end of file
+);
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+CREATE INDEX idx_visits_animal_id ON visits(animal_id);
+CREATE INDEX idx_visits_vet_id ON visits(vet_id);
+CREATE INDEX idx_owners_email ON owners(email);
```

Comment:
- [ ] [OPTIONAL] Since email addresses are not case-sensitive, we might need to look for the emails in a case-insensitive way. There is an option to reflect that in the index by using:
```
CREATE INDEX ON owners ((lower(email)));
```
This will help to get rid potential mistakes or arrors